php/driftphp/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "drift/skeleton",
+    "description": "Skeleton repository ofr DriftPHP",
+    "type": "project",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Marc Morera",
+            "email": "yuhu@mmoreram.com"
+        }
+    ],
+    "require": {
+        "php": "^7.3",
+        "symfony/console": "^5.0.0",
+        "symfony/framework-bundle": "^5.0.0",
+        "symfony/dotenv": "^5.0.0",
+        "symfony/stopwatch": "^5.0.0",
+        "symfony/debug": "^4.4.0",
+
+        "drift/http-kernel": "0.1.*, >=0.1.7",
```

Comment:
this is not very consistent with the constraint above, I suggest using `~` instead of `^`

Reply:
I don't see the inconsistency here. Can you explain a bit more?

Reply:
`^4.4.0` and `0.1.*, >=0.1.7` are quite similar in meaning for composer, but is not very clear (for reader), better use the same semantic

Reply:
Updated,